app/app/stake/page.tsx

```
@@ -486,7 +481,38 @@ function PoolCard({ pool }: { pool: StakePool }) {
 
 /* ── Pool List Section ── */
 
-function PoolList({ pools }: { pools: StakePool[] }) {
+function PoolList({ pools, loading }: { pools: StakePool[]; loading: boolean }) {
+  if (loading) {
+    return (
+      <section id="pools">
+        <div className="mb-4 flex items-center justify-between">
+          <span className="text-[10px] font-medium uppercase tracking-[0.25em] text-[var(--accent)]/60">// available pools</span>
```

Biome (2.4.4)

[error] Wrap comments inside children within braces.

(lint/suspicious/noCommentText)